beta-src/src/utils/state/gameApiSlice/reducers/processMapClick.ts

```diff
@@ -116,16 +116,23 @@ export default function processMapClick(state, clickData) {
       inProgress
     ) {
       const { allowedBorderCrossings } = ordersMeta[orderID];
+
       const canMove = allowedBorderCrossings?.find((border) => {
         const mappedTerritory = TerritoryMap[border.name];
         return Territory[mappedTerritory.territory] === territoryName;
       });
+
       if (canMove) {
         highlightMapTerritoriesBasedOnStatuses(state);
         const command: GameCommand = {
           command: "MOVE",
         };
-        setCommand(state, command, "territoryCommands", territoryName);
+        setCommand(
+          state,
+          command,
+          "territoryCommands",
+          canMove.coastParent.name.toUpperCase(),
```

Comment:
Is there a way to check for the ID of the parent? Currently this doesn't highlight St Petersburg when clicking on the coastal areas.

Reply:
`setCommand` requires a territory enum key like `SAINT_PETERSBURG`, but this code would send `ST. PETERSBURG` since that is the name from the web dip DB. This causes the STP territory not to get highlighted when moving into a coast like Bulgaria or Spain would.